easybuild/easyconfigs/e/ecBuild/ecBuild-3.7.0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'Tarball'
+
+name = 'ecBuild'
+version = '3.7.0'
+
+homepage = 'https://ecbuild.readthedocs.io/'
+
+description = """
+A CMake-based build system, consisting of a collection of CMake macros and
+functions that ease the managing of software build systems """
+
+toolchain = SYSTEM
+
+github_account = 'ecmwf'
+sources = [
+    {
+        'source_urls': [GITHUB_SOURCE],
+        'filename': '%(version)s.tar.gz',
+        'extract_cmd': 'tar -xzf %s --strip-components=1',
+    },
+]
+checksums = ['44ba49b6b628e88bc0145f0cd1fdbdff37fbff2327c581f22a47cd10263a21b6']
+
+buildininstalldir = True
+
+skipsteps = ['install']
```

Comment:
I'm curious what is this doing. `Tarball` will already copy the sources in the build directroy to install directory, isn't it?

Reply:
`buildininstalldir = True` will unpack the sources directly into the installdir, so there is no need for the extra install step.